
Fix for following error on PPC:
> undefined reference to `at::native::DispatchStub<void (*)(c10::ScalarType, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, void const*, c10::ArrayRef<long>, void*), at::native::fft_fill_with_conjugate_symmetry_stub>::VSX'

From https://github.com/pytorch/pytorch/pull/68914

From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Thu, 25 Nov 2021 12:29:50 +0000
Subject: [PATCH] add VSX dispatch for fft_fill_with_conjugate_symmetry_stub

---
 aten/src/ATen/native/mkl/SpectralOps.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/aten/src/ATen/native/mkl/SpectralOps.cpp b/aten/src/ATen/native/mkl/SpectralOps.cpp
index 0625110a04ff..2bd05ca19cac 100644
--- a/aten/src/ATen/native/mkl/SpectralOps.cpp
+++ b/aten/src/ATen/native/mkl/SpectralOps.cpp
@@ -151,6 +151,7 @@ static void _fft_fill_with_conjugate_symmetry_cpu_(
 REGISTER_ARCH_DISPATCH(fft_fill_with_conjugate_symmetry_stub, DEFAULT, &_fft_fill_with_conjugate_symmetry_cpu_)
 REGISTER_AVX2_DISPATCH(fft_fill_with_conjugate_symmetry_stub, &_fft_fill_with_conjugate_symmetry_cpu_)
 REGISTER_AVX512_DISPATCH(fft_fill_with_conjugate_symmetry_stub, &_fft_fill_with_conjugate_symmetry_cpu_)
+REGISTER_VSX_DISPATCH(fft_fill_with_conjugate_symmetry_stub, &_fft_fill_with_conjugate_symmetry_cpu_)
 
 // _out variants can be shared between PocketFFT and MKL
 Tensor& _fft_r2c_mkl_out(const Tensor& self, IntArrayRef dim, int64_t normalization,
